public/index.html

```diff
@@ -3,19 +3,13 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <title>Speechly Home Automation Demo</title>
+    <title>Speechly Browser Client Example</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <h1>Speechly Demo</h1>
-    <p>Check out our spoken language understanding API for developers at <a href="https://speechly.com">speechly.com</a></p>
-    <p>Try saying the following phrases:</p>
```

Comment:
@ottomatias I think without these instructions it is unclear how to use the hosted version of the demo - https://speechly.github.io/browser-client-example/. I personally don't see any issues with having these instructions here - I would assume that anybody who clones the repo can figure out that they are supposed to use their own utterances.

The same applies to the title of the page.

Reply:
Yes. I don't have a strong preference after all. Why I ended up changing them was because when I took some screenshots to show how the browser-client works for documentation it seems funny when I'm not saying anything that are visible in the sample utterances. 
On the other hand yes, if someone is only testing it from the hosted demo it surely needs some instructions. 
Maybe the best way would be to have best of both worlds but that'd require maintaining 2 different versions of the example (one with the utterances and other strictly for cloning) and that's something that someone else than me is going to do so I can't say how much extra work and how much extra help for users that would provide. 
Your call, I think.

Reply:
I think people are smart enough to realise that the sample utterances may not be relevant if they are using a cloned version of the example 🙂 

I'll revert this change and merge the styling fixes 👍